app/src/main/res/values/strings.xml

```diff
@@ -690,30 +690,30 @@
     <string name="suggested_edits_unlock_notification_button">Yes, let’s go</string>
     <string name="suggested_edits_unlock_add_captions_dialog_title">You unlocked the "Add image captions" task</string>
     <string name="suggested_edits_unlock_add_captions_notification_title">You unlocked "Add image captions"</string>
-    <string name="suggested_edits_unlock_add_captions_dialog_message">Thanks for all your contributions! Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
-    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. Since we think you like adding captions to images, we’ve compiled a list of images that are missing captions. Could you help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_add_captions_dialog_message">Congratulations on your third image caption edit! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
+    <string name="suggested_edits_unlock_add_captions_notification_big_text">Suggested edits\n\nCongratulations on your third image caption edit! Since we think you like adding captions to images, we’ve compiled a list of images missing captions. Could you help us improving Wikipedia?</string>
     <string name="suggested_edits_unlock_translate_captions_dialog_title">You unlocked the "Translate image captions" task</string>
     <string name="suggested_edits_unlock_translate_captions_notification_title">You unlocked "Translate image captions"</string>
-    <string name="suggested_edits_unlock_translate_captions_dialog_message">Thanks for all your contributions! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Will you continue to help us improve Wikipedia?</string>
-    <string name="suggested_edits_unlock_translate_captions_notification_big_text">Suggested edits\n\nThanks for all your contributions. You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Will you continue to help us improve Wikipedia?</string>
+    <string name="suggested_edits_unlock_translate_captions_dialog_message">Congratulations on your 15th image caption edit, we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improving Wikipedia?</string>
+    <string name="suggested_edits_unlock_translate_captions_notification_big_text">Suggested edits\n\nCongratulations on your 15th image caption edit, we’re amazed! You’re using the app in multiple languages, so we’ve compiled a list of images needing caption translations. Could you continue to help us improving Wikipedia?</string>
```

Comment:
Let's not put any absolute numbers ("15th") into these strings. There is a 100% chance that these numbers will change later, and they should not be hard-coded into the translations.

Reply:
Make sense. Will update this after getting new strings.

Reply:
Done.